utils/check-dependencies-exist.js

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+'use strict';
+
+const path = require('path');
+const glob = require('glob');
+const colors = require('colors');
+const depCheck = require('depcheck');
+
+const depCheckOptions = {
+  ignoreMatches: [
+    '@testing-library/react',
+    '@storybook/react',
+    'storybook-readme',
+    '@storybook/addon-actions',
+    '@storybook/addon-knobs',
+    'enzyme',
+    'react-dom',
+    'jest-axe',
+    'jest-emotion',
+  ],
+};
+
+const labelMap = {
+  dependencies: {
+    label: 'unused dependency',
+    color: colors.yellow,
+  },
+  devDependencies: {
+    label: 'unused devDependency',
+    color: colors.yellow,
+  },
+  missing: {
+    label: 'missing dependency',
+    color: colors.red,
+  },
+  invalidFiles: {
+    label: 'error',
+    color: colors.red,
+  },
+  invalidDirs: {
+    label: 'error',
+    color: colors.red,
+  },
+};
+
+function formatErrorMessage(errors) {
+  return (
+    `${errors.file}\n` +
+    Object.keys(errors)
+      .map(key => {
+        if (key === 'file') return;
+
+        if (errors[key].constructor === Object) {
+          return Object.keys(errors[key])
+            .map(pkg =>
+              errors[key][pkg]
+                .map(
+                  errorFile =>
+                    `${labelMap[key].color(labelMap[key].label.padEnd(20))}  ${colors.cyan(
+                      pkg
+                    )}  ${colors.dim(errorFile)}`
+                )
+                .join('\n')
+            )
+            .join('\n');
+        } else {
+          return errors[key]
+            .map(
+              error =>
+                `${labelMap[key].color(labelMap[key].label.padEnd(20))}  ${colors.cyan(error)}`
+            )
+            .join('\n');
+        }
+      })
+      .join('\n')
+  );
+}
+
+const searchPath = `${path.resolve(__dirname)}/../modules/**/react`;
+
+glob(searchPath, undefined, function(er, modules) {
```

Comment:
A lot of this logic is searching for the proper folders. Wouldn't it be simpler to update each package to run depcheck as a prebuild step in its own module instead of at the root? Much like we have builds in their respective modules?